.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,41 @@
+name: Continuous integration
+on: [push]
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    env:
+      TOX_PARALLEL_NO_SPINNER: 1
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6.9
+    - name: Update pip
+      run: python -m pip install --upgrade pip
+    - name: Install tox
+      run: python -m pip install tox
+    - name: Checkout git repo
+      uses: actions/checkout@v2
+    - name: Cache the .tox dir
+      uses: actions/cache@v2
+      with:
+        path: .tox
+        key: ${{ runner.os }}-tox-${{ hashFiles('tox.ini', 'requirements*', 'setup.py', 'setup.cfg') }}
+        restore-keys: |
+          ${{ runner.os }}-tox-
+    - name: Run tox
+      run: tox --parallel auto -e checkformatting,lint,tests,coverage
+  frontend:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Cache the node_modules dir
+      uses: actions/cache@v2
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-node_modules-${{ hashFiles('yarn.lock') }}
```

Comment:
Bouncer still uses npm rather than yarn so this should be `package-lock.json`.

Reply:
Fixed